lib/specinfra/backend/docker.rb

```diff
@@ -12,7 +12,9 @@ def initialize
       if image = Specinfra.configuration.docker_image
         @images = []
         @base_image = ::Docker::Image.get(image)
-        create_and_start_container
+
+        env = Specinfra.configuration.docker_env || {}
```

Comment:
Sounds good. I'll patch it up

On Sat, Feb 28, 2015 at 1:35 PM, Gosuke Miyashita
<notifications@github.com> wrote:

>> @@ -12,7 +12,9 @@ def initialize
>>        if image = Specinfra.configuration.docker_image
>>          @images = []
>>          @base_image = ::Docker::Image.get(image)
>> -        create_and_start_container
>> +
>> +        env = Specinfra.configuration.docker_env || {}
>>  I assumed that env was for the running context as opposed to inside the docker container.
> No. `Specinfra.configuration.env` is for defining environment variables in test target machines.
> So it is applicable to docker containers.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/serverspec/specinfra/pull/328/files#r25553071